sunbeam-python/sunbeam/steps/cluster_status.py

```diff
@@ -247,6 +248,12 @@ def _compute_status(self) -> dict:
             try:
                 _status_model = self._get_machines_status(model)
             except ModelNotFoundException as e:
+                if (
+                    is_maas_deployment(self.deployment)
```

Comment:
If the change is specific to MAAS provider, then MaasClusterStatusStep have to be updated instead of base class.